core/src/main/java/greencity/controller/TipsAndTricksTagsController.java

```diff
@@ -0,0 +1,27 @@
+package greencity.controller;
+
+import greencity.service.TipsAndTricksTagsService;
+import java.util.List;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/tipsandtricksTags")
+@AllArgsConstructor
+public class TipsAndTricksTagsController {
+    private final TipsAndTricksTagsService tipsAndTricksTagsService;
+
+    /**
+     * The method which returns all tips & tricks tags.
+     *
+     * @return list of {@link String} (tag's names).
+     */
+    @GetMapping
+    public ResponseEntity<List<String>> findAllTipsAndTricksTags() {
```

Comment:
I think this method can be moved to tipsandtricks controller, as these tags belong to tipsandtricks

Reply:
Removed to the TipsAndTricksController as adviced